src/test/scala/uk/gov/hmrc/govukfrontend/views/components/GovukRadiosSpec.scala

```diff
@@ -19,4 +19,34 @@ package components
 
 import uk.gov.hmrc.govukfrontend.views.html.components._
 
-class GovukRadiosSpec extends TemplateUnitSpec[Radios, GovukRadios]("govukRadios")
+class GovukRadiosSpec extends TemplateUnitSpec[Radios, GovukRadios]("govukRadios") {
+
+  "anti-corruption layer" should {
+    "throw if more than one radio item is checked" in {
```

Comment:
I did a quick browser test and it seems like the last one with checked attribute ends up selected when there are multiple. I think generally it's better to blow up to avoid wrong behaviour, but the way it could go wrong without the require() asking for only one checked item is that the wrong default value could be pre-checked but the user could correct and continue from that, where with the require() we're introducing an unrecoverable error that might prevent people doing stuff. I'm still tempted to say to do it, because otherwise bug might never be found in services if it actually exists anywhere, but I think this would need to be done as a breaking change with a note about it in the changelog

<img width="734" alt="CleanShot 2023-01-06 at 09 09 10@2x" src="https://user-images.githubusercontent.com/100650/210970777-d02937d3-b223-4aec-9461-eb1cf0ec9a62.png">